app/pages/user-reservations/_user-reservations-page.scss

```diff
@@ -25,7 +25,7 @@
 
     color: $black;
     font-weight: $font-weight-bold;
-    font-size: 3rem;
```

Comment:
Optional: should this come from a variable? `_rem-text-size.scss` at least contains `$font-size-h4` which is the closest at `2.4rem`.